docs_src/microservices/configuration/CommonCommandLineOptions.md

```diff
@@ -13,6 +13,9 @@ Can be overridden with [EDGEX_CONF_DIR](./CommonEnvironmentVariables.md#edgex_co
 !!! edgey "EdgeX 3.0"
     The `EDGEX_CONF_DIR` environment variable is replaced by `EDGEX_CONFIG_DIR` in EdgeX 3.0.
 
+!!! edgey "EdgeX 3.0"
```

Comment:
Line 7 above also needs to change and IMO  this `edgey` should be moved up just below that.